content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -16,17 +16,276 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-W3C Trace Context propagation is available for all supported languages. The default trace propagation style for all languages is `datadog,tracecontext`. Envoy and nginx proxies use the default of `tracecontext,datadog`.
+Trace Context propagation is the mechanism of passing tracing information like Trace ID, Span ID, and sampling decisions from one part of a distributed application to another. This enables all traces (and additional telemetry) in a request to be correlated. When automatic instrumentation is enabled, trace context propagation is handled automatically by the tracing library.
```

Comment:
Let's call out the current defaults very clearly here, on their own line with a description of "why"

Reply:
Updated in 89d3d40e48235ff93a8c9f6472ff2c66a587d95e

Reply:
@brett0000FF flagging because I want your review to make sure that the defaults and their "why" is clear to you/users